### ISSUE TITLE
changing default port is not working

### ISSUE THREAD

**Issue by bronstijn-git** [BenjV/SYNO-packages#19]
After creating a backup, uninstalling sickrage (your 'SickChill noarch V1.2' package), installing 'SickChill 1.0 Python 3 noarch' on DSM 6.2.3-25426 Update 2 (DS418) and restoring the backup i cannot change the http-port anymore.
Whenever i change it to let's say 6053 and i restart it still comes up on port 8899.
Anything i missed?

**Comment by BenjV** [comment]
No, it is just not possible to change a port on an application running on Synology without creating problems with Dsm.
Dsm package center uses the port at installation time to register it with Dsm and changing the port by the application itself will mess that up.
To prevent those problems changing the port is blocked.

**Comment by bronstijn-git** [comment]
Thanks, in the previous package it was possible (my sick chill was accessible via port 6053, anyway). But I guess I should enable https, add the certificates and add the port to my nat settings.
And thanks for your installer!

**Comment by BenjV** [comment]
Be aware that application like Sickchill are not build to be security failsave but designed for local use only.
If you make these application accessible from the internet you create a big security risc.

If you want access from the internet you should use a Vpn and not port forwarding.
